src/road_cmd.cpp

```diff
@@ -1113,8 +1113,14 @@ std::tuple<CommandCost, Money> CmdRemoveLongRoad(DoCommandFlag flags, TileIndex
 				cost.AddCost(ret);
 				had_success = true;
 			} else {
-				/* Ownership errors are more important. */
-				if (last_error.GetErrorMessage() != STR_ERROR_OWNED_BY) last_error = ret;
+				/* Some errors are more equal than others. */
```

Comment:
```suggestion
				/* Some errors are more important than others. */
```
The word "important" describes it better than "equal".

Reply:
No. Humor is important!

Reply:
PS: I can be bought off with cookies (or beer) to give me another opinion. Just saying ;)